chapter_13_database_layer_validation.asciidoc

```diff
@@ -325,8 +303,6 @@ and make a note to refactor them out.  But we won't do them straight away,
 because right now our application is in a broken state. We want to get back
 to a working state first.
 
-// JAN: It's preferred to use hardcoded URLs in tests. If we use resolving functions it's easy to change URLs and have passing tests while actually breaking stuff. For example, all the previously sent emails that the old URL.
```

Comment:
I think for this problem is best to have one test to check whether the result of calling `reverse()` is the same as the hardcoded url. And not to use hardcoded URLs anywhere else.